src/notation/view/notationnavigator.h

```diff
@@ -26,7 +26,12 @@
 #include <QMouseEvent>
 #include <QPainter>
 #include <QQuickPaintedItem>
+#include <memory>
+#include <qpainter.h>
+#include <qquickpainteditem.h>
+#include <qwidget.h>
```

Comment:
These includes also seem unnecessary because they duplicate the existing ones; `<QQuickPaintedItem>` and `<qquickpainteditem.h>` are the same for example, and I don't see why `<qwidget.h>` would be necessary here.
(And `<memory>` also becomes unnecessary when you address https://github.com/musescore/MuseScore/pull/19639/files#r1349531903)